modules/calypso-redirects.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+if ( ! class_exists( 'Calypso_Redirects' ) ) :
+
+class Calypso_Redirects {
+
+	public function __construct() {
+		if ( ! get_option( 'jetpack_calypso_redirects_active' ) || ! is_admin() ) {
```

Comment:
Just curious, why did you opt to make this controlled by an option and not by a filter? Wouldn't it be better if this were activated via a filter, so that option can't be ported over to another host if the site owner decides to move their database somewhere else or uses a cloning plugin?